### PR TITLE
style: add category property to commands

### PR DESCRIPTION
**Pull request by nriviere-gitguardian**
## Context

The display of the commands was not standardized: some commands were not prepended by GitGuardian, and the others were using _gitguardian_ instead of _GitGuardian_.

CleanShot 2024-11-13 at 11 26 57@2x

## What has been done

Update `package.json` to group GitGuardian commands into the `GitGuardian` category.

## Validation

Open the command palette and verify that GitGuardian commands are prepended by _GitGuardian:_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,35 +55,43 @@
...
       {
         "command": "gitguardian.authenticate",
-        "title": "gitguardian: Authenticate"
+        "title": "Authenticate",
+        "category": "GitGuardian"
       },
       {
         "command": "gitguardian.logout",
-        "title": "gitguardian: logout"
+        "title": "logout",
```
Comment (sevbch) [inline package.json:73]: Just to be consistent with all titles starting with a capital letter
```suggestion
        "title": "Logout",
```

**Review by sevbch — COMMENTED** [review]
So you've added `"category": "GitGuardian"` for all commands, but in your screenshot, where is the `GitGuardian:` prefix coming from? Isn't going to be doubled? (or is it already coming from this PR?)

**Review by salome-voltz — APPROVED** [review]
Thank you ! :pray:

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (nriviere-gitguardian) [reply package.json]: Sure!

**Comment by nriviere-gitguardian** [comment]
> So you've added `"category": "GitGuardian"` for all commands, but in your screenshot, where is the `GitGuardian:` prefix coming from? Isn't going to be doubled? (or is it already coming from this PR?)

It's coming from the `views` property: the VS Code automatically adds commands to open the views and prepend the command by the name of the extension, `GitGuardian` in this case.

I've added a commit to fix some case issues in the views names.

**Pull request merged**